commands.go

```diff
@@ -183,6 +201,92 @@ func (cmd *MonitorStatus) SetValue(result *Result) error {
 	return nil
 }
 
+// FreezeFrame represents a command that checks the DTC that caused required 
+// freeze frame data storage
+// 
+// Min 0x0000
+// Max 0xFFFF
+type FreezeFrame struct {
+	BaseCommand
+	FrameData string
```

Comment:
The data that the commands produce should be decoded and not the raw representation. 

If we [look at the formula](https://en.wikipedia.org/wiki/OBD-II_PIDs#Service_02) needed to decode the result of this command, we see that it says "BCD encoded. [Decoded as in service 3](https://en.wikipedia.org/wiki/OBD-II_PIDs#Service_03)". 

Basically, there are 3 parts that needs to be decoded from the 2 bytes, instead of a single raw string.